### DISCUSSION TITLE
Im not receive my benefit after reverifying Student Dev

### DISCUSSION THREAD

**Discussion by pasya1912** [community/community#108594] [GitHub Education]
### Hi everyone,

I have reverify my Student status, but i dont receive my benefit yet, why ?

**Comment by kanduru-abhiram** [comment]
Hi @pasya1912 , please check whether your Application request is approved and the background color of Approved in green.

![image](https://github.com/community/community/assets/85011280/243cdc64-5fc7-4443-aeac-c36224d65c0b)

And wait till the background color of Approved turns to Violet, once turned, you'll get the access to all the benefits of Student Developer pack.

![image](https://github.com/community/community/assets/85011280/8809cebe-d954-4843-9e2f-4ef38cc5b082)

Please accept this as a solution if helped, thanks.
